RELEASE_NOTES

```diff
@@ -3,6 +3,37 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.5.0 (December 17th 2015)
+---------------------------
+
+feature + bugfix release
+- add support for IBM XL compilers on Power7 and PowerPC (BlueGene) (#1470)
```

Comment:
would be nice to document which easyconfig names correspond to that; other than that, it all LGTM!